recipe/meta.yaml

```diff
@@ -43,7 +42,7 @@ requirements:
   build:
     - toolchain
     - cmake
-    - libsodium
+    - libsodium  # [not (win and py27)]
```

Comment:
We are missigne.the with libsodium=ON apparently which I will only add on unix and (windows + py>27).